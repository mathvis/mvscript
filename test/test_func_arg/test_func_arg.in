func a(b: int, c: string) int;
