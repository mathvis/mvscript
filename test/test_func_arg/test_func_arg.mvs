[fwd] func a(b: int, c: string) int;
