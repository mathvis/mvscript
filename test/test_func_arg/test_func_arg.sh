#!/bin/bash
helix test_func_arg.mvs test_func_arg.out --vsplit
