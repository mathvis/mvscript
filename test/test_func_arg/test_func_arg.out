Decl (FunctionDef (FunctionIdentifier "a") [(VarIdentifier "b",int),(VarIdentifier "c",string)] int Nothing)
